src/python/grpcio/grpc/experimental/aio/_call.py

```diff
@@ -394,21 +420,29 @@ def _metadata_sent_observer(self):
                 self.cancel()
             await self._raise_for_status()
 
-    async def done_writing(self) -> None:
-        """Implementation of done_writing is idempotent."""
+    async def _done_writing(self) -> None:
         if self.done():
             # If the RPC is finished, do nothing.
             return
-        if not self._done_writing:
+        if not self._done_writing_flag:
             # If the done writing is not sent before, try to send it.
-            self._done_writing = True
+            self._done_writing_flag = True
             try:
                 await self._cython_call.send_receive_close()
             except asyncio.CancelledError:
                 if not self.cancelled():
                     self.cancel()
                 await self._raise_for_status()
 
+    async def write(self, request: RequestType) -> None:
+        self._raise_for_different_style(_APIStyle.READER_WRITER)
+        await self._write(request)
+
+    async def done_writing(self) -> None:
+        """Implementation of done_writing is idempotent."""
```

Comment:
Let's not use the word "implementation." How about just "This method is idempotent"?

Reply:
The original intend is to annotate the expectation of this method. It felt like too short to be a docstring, and not suitable as a comment.

Expanded to:
```python
async def done_writing(self) -> None:
        """Signal peer that client is done writing.

        This method is idempotent.
        """
```